client/app/iplb/configuration/iplb-configuration.html

```diff
@@ -1,27 +1,40 @@
+    <cui-page-content-title data-text="'iplb_configuration_title' | translate"></cui-page-content-title>
     <p data-translate="iplb_configuration_intro"></p>
-
-    <button type="button"
-        class="oui-button oui-button_secondary"
-        data-translate="iplb_configuration_action_apply"
-        data-ng-click="ctrl.applyChanges()"
-        data-ng-disabled="!ctrl.selectedZones.length"></button>
-
-    <oui-spinner data-ng-show="ctrl.zones.loading"></oui-spinner>
-    <oui-table data-ng-hide="ctrl.zones.loading"
-        rows="ctrl.zones.data"
-        on-selection-change="ctrl.onSelectionChange($selection)">
-        <column title="'iplb_configuration_datacenter' | translate" property="name" sortable="asc"></column>
-        <column title="'iplb_configuration_progress' | translate"
-            sortable
-            class="oui-table__cell_component">
+    <oui-datagrid data-rows="ctrl.zones.data">
+        <extra-top>
+            <button type="button"
+                class="oui-button oui-button_secondary"
+                data-translate="iplb_configuration_action_apply"
+                data-ng-click="ctrl.applyChanges()"
+                data-ng-disabled="!ctrl.selectedZones.length"></button>
+        </extra-top>
+        <oui-column>
+            <oui-checkbox
+                name="zones{{$row.id}}"
+                id="zones{{$row.id}}"
+                data-on-change="ctrl.onSelectionChange($row)">
+            </oui-checkbox>
+        </oui-column>
+        <oui-column data-title="'iplb_configuration_datacenter' | translate" data-property="name" data-sortable="asc"></oui-column>
+        <oui-column data-title="'iplb_configuration_progress' | translate" data-sortable>
             <cui-task-progress data-ng-show="$row.task" data-task="$row.task"></cui-task-progress>
-        </column>
-        <column title="'iplb_configuration_status' | translate" template="ctrl.statusTemplate()" class="oui-table__cell_action"></column>
-        <column title="'iplb_configuration_action' | translate"
-            class="oui-table__cell_action oui-table__cell_action-menu"
-            template="ctrl.actionTemplate()">
-        </column>
-    </oui-table>
+        </oui-column>
+        <oui-column data-title="'iplb_configuration_status' | translate">
+            <span data-ng-if="$row.changes === 0" data-translate-attr="{ title: 'iplb_configuration_changes_0' }">
+                <cui-status-icon data-type="success"></cui-status-icon>
+            </span>
+            <span data-ng-if="$row.changes === 1" data-translate-attr="{ title: 'iplb_configuration_changes_1' }">
+                <cui-status-icon data-type="warning"></cui-status-icon>
+            </span>
+            <span data-ng-if="$row.changes > 1" data-translate-attr="{ title: 'iplb_configuration_changes_count' }"
+                data-translate-values="{ count: $row.changes }">
+                <cui-status-icon data-type="warning"></cui-status-icon>
+            </span>
```

Comment:
I'm not sure but I think `ngPluralize` could be used here

Reply:
the message is displayed as "{{count}} changes". It has taken care in the translation file.